MHVLib/Methods/MHVBlobDownloadRequest.m

```diff
@@ -27,4 +29,10 @@ - (instancetype)initWithURL:(NSURL *)url
     return self;
 }
 
+- (NSString *)getCacheKey
+{
+    [NSException throwNotImpl];
```

Comment:
Whoa there cowboy!!! Let's not throw an exception here, lets just log it for debugging purposes.